src/app.ts

```diff
@@ -301,8 +302,16 @@ export class CeramicAnchorApp {
       // await anchorService.garbageCollectPinnedStreams().catch((error) => {
       //   logger.err(`Error when garbage collecting pinned streams: ${error}`)
       // })
+
+      return success
+    }
+
+
+    const cbAfterEmptyBatch = () => {
+      logger.imp('No batches available. Continual anchoring is shutting down.')
+      process.exit(0)
```

Comment:
Do we usually call exit directly in the process like this?

Reply:
Yes I'm following the conventions in this file (ex. https://github.com/ceramicnetwork/ceramic-anchor-service/blob/6a73c563c8fdecfe04f2064cd632c107e3dd9fd1/src/app.ts#L262C5-L262C19)